chitchat/src/server.rs

```diff
@@ -144,18 +146,23 @@ pub async fn spawn_chitchat(
     let chitchat_arc = Arc::new(Mutex::new(chitchat));
     let chitchat_arc_clone = chitchat_arc.clone();
 
+    let (termination_watcher_sender, termination_watcher) = watch::channel(None);
```

Comment:
```suggestion
    let (termination_watcher_sender, termination_watcher) = watch::channel(None);
```

I think that codebase adopts the _tx for sender and _rx for receiver naming scheme.
It is confusing, but common in rust.